lib/cbus_elixir_web/templates/page/index.html.eex

```diff
@@ -12,19 +12,23 @@
 
         
         <h2 class="title myriad">This Month's Speakers</h2>
-        <ul class="list">
-            <%= Enum.map(@next_meeting.speakers, fn(s) -> %>
-                <li>
-                    <a href="<%= s.url %>">
-                        <img class="avatar" src='<%= gravatar_url(s.email) %>', alt="Speaker Image" />
-                        <div class="details">
-                            <div class="name"><%= s.name %></div>
-                            <div class="talk-name"><%= s.title %></div>
-                        </div>
-                    </a>
-                </li>
-            <% end) %>
-        </ul>
+        <%= if Enum.empty?(@next_meeting.speakers) do %>
```

Comment:
You could probably just make this do `Enum.any?` and drop the else statement.

Looks fine to me though

Reply:
Thanks @Kartstig 
`.any?` would just basically flip what I have. Mine is from the approach that nothing is there first. Either way is fine with me. Not sure either one is better than the other, right?